### ISSUE TITLE
Around 5 billion dropped entries on P2

### ISSUE THREAD

**Issue by cayolblake** [madMAx43v3r/chia-plotter#583]
Hello,

I'm getting the following...

```
[P2] Table 7 scan took 64.342 sec
[P2] Table 7 rewrite took 101.659 sec, dropped 0 entries (0 %)
[P2] Table 6 scan took 72.0408 sec
[P2] Table 6 rewrite took 197.033 sec, dropped 581589561 entries (13.5494 %)
[P2] Table 5 scan took 76.9387 sec
[P2] Table 5 rewrite took 187.423 sec, dropped 762356078 entries (17.7554 %)
[P2] Table 4 scan took 73.3449 sec
[P2] Table 4 rewrite took 186.928 sec, dropped 829225236 entries (19.3097 %)
[P2] Table 3 scan took 73.6602 sec
[P2] Table 3 rewrite took 193.011 sec, dropped 855274573 entries (19.9149 %)
[P2] Table 2 scan took 73.0457 sec
[P2] Table 2 rewrite took 185.298 sec, dropped 865753587 entries (20.1576 %)
Phase 2 took 1490.75 sec
```

What does it mean that these short to 5 billion dropped entries mean?

**Comment by madMAx43v3r** [comment]
you mean in total? that's normal

**Comment by number435398** [comment]
I can totally confirm that's normal.  Though I don't know why.  It is normal, no matter how you slice it.

**Comment by cayolblake** [comment]
@madMAx43v3r I've seen Bram's video analyzing your plotter and stating that there are around %0.1 lost entries and that the generated plots aren't bit for bit identical to stock plotter.

Are these ones he was talking about?

How does they affect farming?

Fixing these on the milestone?

**Comment by madMAx43v3r** [comment]
nope, those are normal dropped entries. it's 0.01% which are dropped in addition at most, your plot will the smaller too, in the end no difference

**Comment by cayolblake** [comment]
@madMAx43v3r but the progress output shows it's dropping up to %20 which is extremely confusing!

Can you please clarify from what you can see in my initial post.

Thanks.

**Comment by VertiHydro** [comment]
> @madMAx43v3r but the progress output shows it's dropping up to %20 which is extremely confusing!
> 
> Can you please clarify from what you can see in my initial post.
> 
> Thanks.

This is not the same thing as the 0.01% dropped entries in phase 1. This 20% or whatever is a normal result of Phase 2, which aims to remove unnecessary entries in the plot tables. From [this](https://medium.com/geekculture/chia-network-plotting-optimization-7f0d6380eded) excellent article:
"The purpose of the backpropagation step is to remove data that is not useful for finding proofs. During this phase, all entries which do not form part of the matching condition in the next table, are dropped."

**Comment by Nitrooo** [comment]
Is the "0.01% drop" going to be fixed anyway? Despite being negligible in terms of plot quality, Bram himself talks about it as an undesirable effect of this plotter.

**Comment by Nitrooo** [comment]
@madMAx43v3r Is the "0.01% drop" going to be fixed anyway? Despite being negligible in terms of plot quality, Bram himself talks about it as an undesirable effect of this plotter.

**Comment by number435398** [comment]
> 
> 
> @madMAx43v3r Is the "0.01% drop" going to be fixed anyway? Despite being negligible in terms of plot quality, Bram himself talks about it as an undesirable effect of this plotter.

How is the drop even material to anything of concern?

**Comment by madMAx43v3r** [comment]
> @madMAx43v3r Is the "0.01% drop" going to be fixed anyway? Despite being negligible in terms of plot quality, Bram himself talks about it as an undesirable effect of this plotter.

No plans as of today, it'll slow phase 1 down and increase tmp 1 + 2 space needed by one byte per entry.

**Comment by cayolblake** [comment]
> > @madMAx43v3r Is the "0.01% drop" going to be fixed anyway? Despite being negligible in terms of plot quality, Bram himself talks about it as an undesirable effect of this plotter.
> 
> 
> 
> No plans as of today, it'll slow phase 1 down and increase tmp 1 + 2 space needed by one byte per entry.

Wow! Then make it optional if it relates to performance or required temp storage :/

**Comment by Nitrooo** [comment]
> Wow! Then make it optional if it relates to performance or required temp storage :/

That's the way to go, in my opinion.

**Comment by number435398** [comment]
I still fail to see how this issue is material to anything of concern. There are far more significant issues in this forum than this. Seriously, people are getting 23 minute plot times, smaller plots, and you're saying that something that, you have yet to demonstrate has any measurable impact on those things, is a problem?

**Comment by madMAx43v3r** [comment]
it's a cosmetic / emotional issue, maybe when we add it to official chia repo, it's not possible to have a dynamic switch, has to be hardcoded either way

**Comment by cayolblake** [comment]
If that's so, I'd say we shouldn't compromise performance/storage utilization.

**Comment by Jacek-ghub** [comment]
> No plans as of today, it'll slow phase 1 down and increase tmp 1 + 2 space needed by one byte per entry.


Just out of curiosity, are there any estimates of what "slow phase 1 down" and "increase tmp 1 + 2 space" values might be?
